kratos/utilities/element_size_calculator.cpp

```diff
@@ -383,6 +605,159 @@ double ElementSizeCalculator<3,4>::MinimumElementSizeDerivative(
     KRATOS_CATCH("");
 }
 
+// Tetrahedra3D4 version.
+template<>
+double ElementSizeCalculator<3,10>::MinimumElementSize(const Geometry<Node<3> >& rGeometry)
+{
+
+    /* Calculate distances between each node and the opposite face */
+    const double x10 = rGeometry[1].X() - rGeometry[0].X();
+    const double y10 = rGeometry[1].Y() - rGeometry[0].Y();
+    const double z10 = rGeometry[1].Z() - rGeometry[0].Z();
+
+    const double x20 = rGeometry[2].X() - rGeometry[0].X();
+    const double y20 = rGeometry[2].Y() - rGeometry[0].Y();
+    const double z20 = rGeometry[2].Z() - rGeometry[0].Z();
+
+    const double x30 = rGeometry[3].X() - rGeometry[0].X();
+    const double y30 = rGeometry[3].Y() - rGeometry[0].Y();
+    const double z30 = rGeometry[3].Z() - rGeometry[0].Z();
+
+    // face 123
+    double nx = (y30-y10)*(z20-z10) - (z30-z10)*(y20-y10);
+    double ny = (z30-z10)*(x20-x10) - (x30-x10)*(z20-z10);
```

Comment:
Const when possible

Reply:
These variables cannot be `const` since are modified below

Reply:
Okay, my main complain was about the push_back

Reply:
Yeah, I know, but it is a test, so it is run only once. The rest of the element size tests are created in a similar way. Are you sure we will appreciate the enhancement?